src/styles/components/_check-list.scss

```diff
@@ -0,0 +1,30 @@
+@font-face {
+  font-family: "gcds-icons";
+  src: url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.eot");
+  src: url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.eot#iefix")
+      format("embedded-opentype"), url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.ttf")
+      format("truetype"),
+    url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.woff")
+      format("woff"), url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.svg")
+      format("svg");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+.check-list {
+  > li {
+    position: relative;
+    max-width: 65ch;
+    counter-increment: item;
+    padding-inline-start: var(--gcds-spacing-300);
+
+    &:before {
+      font-family: "gcds-icons" !important;
```

Comment:
Note to update this to use the icon font family token once the icon change has been released.